src/modules/project/funding/hooks/useFundingAPI.ts

```diff
@@ -120,15 +120,22 @@ export const useFundingAPI = () => {
       const keyPair = generatePrivatePublicKeyPair()
       setKeyPair(keyPair)
 
-      if (input?.paymentsInput?.onChainSwap?.boltz) {
-        input.paymentsInput.onChainSwap.boltz.swapPublicKey = keyPair.publicKey.toString('hex')
+      const { geyserTipPercent } = formState
+
+      const finalInput = {
+        ...input,
+        geyserTipPercentage: geyserTipPercent > 0 ? geyserTipPercent : undefined,
+      }
+
+      if (finalInput?.paymentsInput?.onChainSwap?.boltz) {
+        finalInput.paymentsInput.onChainSwap.boltz.swapPublicKey = keyPair.publicKey.toString('hex')
```

Comment:
Maybe update this in the state level in src/modules/project/funding/state/fundingContributionCreateInputAtom.ts